### ISSUE TITLE
No topic and broker is shown

### ISSUE THREAD

**Issue by pajain30** [yahoo/kafka-manager#683]
Hi

in the application.conf file i have given zookeeper host and the exposed port of the zookeeper and it is not showing any error , it shows that its connected successfully.

but on the UI its shows 0 topic and 0 brokers although kafka CLI shows the topic and other information.

can you help me if i need to configure anything else also?

Thanks

**Comment by gustavoveronese** [comment]
same herer

**Comment by ap2809** [comment]
Same here...

**Comment by davehouser1** [comment]
Yup same here, does CMAK look in a specific directories for brokers? I adjusted my Kakfa properties to use a custom directory, is that not supported?
